src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests.csproj

```diff
@@ -92,7 +100,9 @@
     </None>
     <None Include="project.json" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
```

Comment:
@davkean. Why is that? The threading mock needs the Dispatcher class. None of this is VS specific.